snudown.h

```diff
@@ -0,0 +1,118 @@
+/* snudown.h - header for snudown */
```

Comment:
Agree with @pwildani's approach. I really should have put all the shared rendering logic into a shared `renderers.c` when I first wrote the fuzzing code 😬 . 

Also, I don't believe the fuzzer is invoked when calling `python setup test`. The fuzzing process has a number of extra dependencies (AFL, CMake, the custom Google gumbo bits...) that aren't needed for functional tests. It also doesn't really fit into what people would normally expect `python setup.py test` to do (you need to run it for multiple days to get trustworthy results.)

I need to update the docs on how to run the fuzzing though, feel free to ping me on Slack about getting it working